src/main/docs/guide/ioc/annotationMetadata.adoc

```diff
@@ -61,13 +63,14 @@ If you do not have control over the annotation then another approach is to use a
 
 NOTE: Because `AnnotationMapper` implementations need to be on the annotation processor classpath they should generally be in a project that includes few external dependencies to avoid polluting the annotation processor classpath.
 
-As an example the the `AnnotationMapper` that maps the `javax.annotation.security.PermitAll` standard Java annotation to the internal Micronaut `Secured` annotation looks like the following:
+As an example the  `AnnotationMapper` that maps the `javax.annotation.security.PermitAll` standard Java annotation to the internal Micronaut `Secured` annotation looks like the following:
 
 .PermitAllAnnotationMapper Mapper Example
 [source,java]
 ----
 include::security/src/main/java/io/micronaut/security/annotation/PermitAllAnnotationMapper.java[tag=clazz]
 ----
+// TODO above link is broken. Source is in an external project. How do we fix?
```

Comment:
Either use another annotation mapper in core as an example or remove this section